build-sdist/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Build source distribution'
+description: 'Build a Python source distribution using build and upload the resulting artifact'
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      name: Install Python
+      with:
+        python-version: '3.8'
+
+    - name: Install build tools
+      shell: 'bash'
+      run: python -m pip install build

```

Comment:
```suggestion
      run: python -m pip install build "twine>=3.3"

```

Reply:
This step does not do any publishing, so I don't think twine is necessary here; the `publish` step takes care of that.